### ISSUE TITLE
Is there a Code Style Guide

### ISSUE THREAD

**Issue by somkun** [Kunzisoft/KeePassDX#184]
I'm working on a PR for KeePassDX, and I auto-format code (because I don't trust myself to keep standard formatting), but I've noticed my formatter changing a line or two that I didn't write. Before I send a PR with a few formatting clean ups, I wanted to know if there's a specific set of formatting rules I should be applying.

**Comment by J-Jamet** [comment]
Not really, but you're right, it would be nice to make rules for formatting. I just made lot of changes from the KeePassDroid project that was on Eclipse, so there are tabbing issues, and so on. If you have rules that you want to apply, you can tell me which ones, to see if we can apply them to the whole project.

**Comment by somkun** [comment]
As an easy first step, we can adopt [Google's style guide](https://github.com/google/styleguidel). It has preexisting definitions for IntelliJ, Android Studio, Eclipse, and VSCode. Given the ubiquity of Google's developers, I'd be willing to bet that there are existing vim and emacs presets as well.
